core/math/transform_2d.cpp

```diff
@@ -277,5 +277,7 @@ Transform2D Transform2D::interpolate_with(const Transform2D &p_transform, real_t
 }
 
 Transform2D::operator String() const {
-	return String(String() + elements[0] + ", " + elements[1] + ", " + elements[2]);
+	return "[X: " + elements[0].operator String() +
+		   ", Y: " + elements[1].operator String() +
+		   ", O: " + elements[2].operator String() + "]";
```

Comment:
You seem to have space indentation/alignment here (and in other classes). It's annoying that `clang-format` allows it even though we use tabs only.

Does it work with a double-tab continuation indent instead?

Reply:
@akien-mga It doesn't. `clang-format` automatically changes it back to the current state.